src/application-navigation/terra-application-navigation/clinical-lowlight-theme/ApplicationNavigation.module.scss

```diff
@@ -0,0 +1,9 @@
+:local {
+  .clinical-lowlight-theme {
+    --terra-application-navigation-drawer-background-color: #222a2e;
+    --terra-application-navigation-drawer-default-width: calc(100% - 3.571rem);
+    --terra-application-navigation-drawer-medium-width: 20rem;
+    --terra-application-navigation-content-layout-background-color: #1c2326;
+    --terra-application-navigation-content-layout-box-shadow: 7px 0 7px 10px rgba(0, 0, 0, 0.4);
+  }
```

Comment:
All of these themed vars need to be updated with the prefix: `--terra-application-application-navigation`

Reply:
wdio tests would have caught this 🤔